data/tosca/yorc-slurm-types.yml

```diff
@@ -90,6 +92,14 @@ data_types:
         required: false
         entry_schema:
           type: string
+      in_script_options:
```

Comment:
Shouldn't the `template_version` above be increased ? Wondering if we don't change it, if a yorc upgrade would store this new definition or would just keep the previous definition stored in consul

Reply:
Yes could be changed to 1.2.1 or 1.3.0 but anyway this do not correspond to what we now have in a4c as it is now always versioned along with a4c